example/themes/original/src/components/PhoneInputNumber/index.tsx

```diff
@@ -59,7 +59,7 @@ export const PhoneInputNumber = (props: PhoneInputParams) => {
 	}
 
 	useEffect(() => {
-		if ((defaultValue && userphoneNumber) || defaultValue === undefined || defaultValue === '') {
+		if ((defaultValue && userphoneNumber) || defaultValue === undefined || defaultValue === '' || defaultValue === null) {
```

Comment:
Please only use the validation `!defaultValue` instead `defaultValue === undefined || defaultValue === '' || defaultValue === null`

Reply:
Okay, I've fixed